src/Tgstation.Server.Api/Models/DiscordConnectionStringBuilder.cs

```diff
@@ -18,6 +18,11 @@ public sealed class DiscordConnectionStringBuilder : ChatConnectionStringBuilder
 		/// <remarks>See https://discordapp.com/developers/docs/topics/oauth2#bots</remarks>
 		public string? BotToken { get; set; }
 
+		/// <summary>
+		/// Boolean to enable based mode (Will auto reply with 'based on the hardware thats installed in it' to anyone saying 'based on what')
```

Comment:
```suggestion
		/// <see cref="bool"/> to enable based mode. Will auto reply with a youtube link to a video that says "based on the hardware that's installed in it" to anyone saying 'based on what?' case-insensitive.
```

Here and in the field doc.

Alternatively, the comment doesn't need to be so descriptive. Could just be `Whether or not the "Based on what?" meme is enabled.`